src/main/java/com/faforever/client/main/event/NavigationItem.java

```diff
@@ -1,16 +1,20 @@
 package com.faforever.client.main.event;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
 import java.util.HashMap;
 
+@RequiredArgsConstructor
 public enum NavigationItem {
```

Comment:
This class also looks like a candidate for `@Value` (see above)

Reply:
Not possible on enums